runtime/vm/BufferWriter.hpp

```diff
@@ -90,12 +91,13 @@ class VM_BufferWriter {
 
 	public:
 
-	VM_BufferWriter(U_8 *buffer, UDATA size)
+	VM_BufferWriter(J9PortLibrary *privatePortLibrary, U_8 *buffer, UDATA size)
```

Comment:
Please don't use the special name `privatePortLibrary`:
```c
	VM_BufferWriter(J9PortLibrary *portLibrary, U_8 *buffer, UDATA size)
		: _portLibrary(portLibrary)
		, _buffer(buffer)
```

Reply:
Fixed.